src/System.IO.FileSystem.Primitives/src/System.IO.FileSystem.Primitives.csproj

```diff
@@ -4,24 +4,31 @@
   <PropertyGroup>
     <ProjectGuid>{6C05678E-394C-4CFF-B453-A18E28C8F2C3}</ProjectGuid>
     <AssemblyName>System.IO.FileSystem.Primitives</AssemblyName>
-    <IsPartialFacadeAssembly Condition="'$(TargetGroup)' == 'net46'">true</IsPartialFacadeAssembly>
+    <IsPartialFacadeAssembly Condition="'$(TargetGroup)' != 'netcore50aot'">true</IsPartialFacadeAssembly>
```

Comment:
We cannot remove this netcore50 build. This package was portable and now is not portable to .net native.